docs/tutorials/index.md

```diff
@@ -1,6 +1,33 @@
 # Introduction
```

Comment:
```suggestion
```
This heading gives the TOC in the rendered version an extra layer, with all of the tutorials listed under the Intro entry.

Reply:
I have an idea for a cleaning formatting of the TOC navigation @JessicaS11 and @naclomi I'll open a separate PR after this one is merged with a proposal